CHANGELOG.rst

```diff
@@ -21,6 +21,8 @@ Changed
 Fixed
 -----
 - Fixed a factor of 1/2 missing in ScatteringFitComponentXTables
+- Bugfix related to Numpy 1.24.0. Strict array creation with dtype=object is needed
+for ragged arrays. (#880 & #881)
```

Comment:
Don't think Sphinx will display this entry nicely without these two leading spaces

```suggestion
  for ragged arrays. (#880 & #881)
```

Reply:
Yes you are correct :) One day I'll figure out how to get things to render in sphinx correctly on the first time.